docs/basic_training/operators.md

```diff
@@ -121,7 +121,9 @@ c1 = Channel.of(1, 2, 3)
 c2 = Channel.of('a', 'b')
 c3 = Channel.of('z')
 
-c1.mix(c2, c3).view()
+c1
+    .mix(c2, c3)
+    .view()
```

Comment:
I find this a bit difficult to read, as `c1` is such a short channel name. How about renaming `c1` to be `my_channel_1` or something that's longer?

Reply:
This one was also bothering me. I was in pain 😆